src/com/massivecraft/factions/engine/EngineCombat.java

```diff
@@ -82,6 +85,38 @@ public void canCombatDamageHappen(PotionSplashEvent event)
 			event.setIntensity(affectedEntity, 0.0);
 		}
 	}
+	
+	@SuppressWarnings("deprecation")
+	@EventHandler(priority = EventPriority.NORMAL, ignoreCancelled = true)
+	public void canCombatDamageHappen(AreaEffectCloudApplyEvent event)
+	{
+		// If a harmful potion effect cloud is present ...
+		if (!MUtil.isHarmfulPotion(event.getEntity().getBasePotionData().getType().getEffectType())) return;
```

Comment:
Actually, @Markehme,  Cayorion changed his mind on the spaces around the !. So the format was correct in this case. ^^